jetty-http2/http2-hpack/src/main/java/org/eclipse/jetty/http2/hpack/HpackEncoder.java

```
@@ -200,7 +200,10 @@ else if (metadata.isResponse())
                         encode(buffer, TE_TRAILERS);
                     continue;
                 }
-                if (hopHeaders != null && hopHeaders.contains(StringUtil.asciiToLowerCase(field.getName())))
+                String name = StringUtil.asciiToLowerCase(field.getName());
+                if (hopHeaders != null && hopHeaders.contains(name))
+                    continue;
+                if (name.startsWith(":"))
```

I think we decided that failure here rather than ignore would be better